// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/conformance/parser/ecmascript5/IndexSignatures/parserIndexSignature11.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,8 +1,9 @@
 
 
 //// [parserIndexSignature11.d.ts]
 interface I {
+    [p]: any;
     [p1: string]: any;
     [p2: string, p3: number]: any;
 }
 
